core/src/main/java/com/netflix/conductor/core/execution/mapper/SubWorkflowTaskMapper.java

```diff
@@ -116,7 +116,7 @@ private Integer getSubWorkflowVersion(Map<String, Object> resolvedParams, String
                 .map(Object::toString)
                 .map(Integer::parseInt)
                 .orElseGet(
-                        () -> metadataDAO.getLatest(subWorkflowName)
+                        () -> ((Optional<WorkflowDef>)metadataDAO.getLatest(subWorkflowName))
```

Comment:
Is it really `WorkflowDef` here or an extension?

Reply:
Actually here it does not matter, since this code will only use the parent class (in this case WorkflowDef). Tried to introduce the generics at model level but i faced a limitation coming from Protogen - it cannot map generics :(